blueoil/converter/templates/Makefile

```diff
@@ -116,19 +116,19 @@ clean:
 	-$(RM) $(OBJ)
 
 lm_x86:           CXX = g++
-lm_x86:           FLAGS += $(INCLUDES) -O3 -std=c++14 -DUSE_PNG -pthread -g
+lm_x86:           FLAGS += $(INCLUDES) -O3 -std=c++14 -DUSE_PNG -pthread -g -DFUNC_TIME_MEASUREMENT
```

Comment:
maybe we should place all `-D` flags next to each other.
`-DUSE_PNG -DFUNC_TIME_MEASUREMENT -pthread -g`

Reply:
Maybe it's better to place `-D` like you pointed out.
That will be modified in another PR.